pkg/kubelet/kubelet.go

```diff
@@ -1776,6 +1777,51 @@ func (kl *Kubelet) recordNodeOnlineEvent() {
 	kl.recorder.Eventf(kl.nodeRef, "online", "Node %s is now online", kl.hostname)
 }
 
+// Returns the most recent sys oom event from cadvisor.
+// Returns nil if no sys oom events were observed.
+func (kl *Kubelet) getRecentSysOOMEvent() (*cadvisorApi.OomEventData, util.Time, error) {
```

Comment:
WDYT of putting this in the cAdvisor subpackage and exposing a "RecentSysOOMEvent()". Trying to minimize code in kubelet.go :)